app/policies/patient_policy.rb

```diff
@@ -0,0 +1,29 @@
+class PatientPolicy < ApplicationPolicy
```

Comment:
We can extract this as another class and have patient also inherit from it. 
Semantically, there should be no relation between patient, district and facility policies.